<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuralNetworkSimulator</class>
 <widget class="QMainWindow" name="NeuralNetworkSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeuralNetworkSimulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>855</width>
      <height>483</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Simulation</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Nr of Iterations</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nrOfIterations">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Eta</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="eta">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>MSE Error</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mseError">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Momentum</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="momentum">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputFile">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Neuron activation</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="activationTemp"/>
       </item>
       <item>
        <widget class="QPushButton" name="trainningButton">
         <property name="text">
          <string>Training</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="outputDisplay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>381</width>
         <height>481</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>381</width>
         <height>481</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Output of the network simulation or run.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Dim of Input Rectangle</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>-4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dimInputRectangle">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Distance  between input rect</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="distInputRect">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Dim of Connection Rectangle</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dimConnectionRect">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Distance between conn rect</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="distConnRect">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Map first level</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nrOfMapsFirstLevel">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Map Second Level</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nrOfMapsSecondLevel">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Neural Network</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="networkType"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nr of Inputs</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nrOfInputs">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nr or Outputs</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nrOfOutputs">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Nr of Hidden Levels</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nrOfHiddenLevels">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Nr of Hidden 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nrOfHidden1">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Nr of Hidden 2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nrOfHidden2">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>networkType</tabstop>
  <tabstop>nrOfInputs</tabstop>
  <tabstop>nrOfOutputs</tabstop>
  <tabstop>nrOfHiddenLevels</tabstop>
  <tabstop>nrOfHidden1</tabstop>
  <tabstop>nrOfHidden2</tabstop>
  <tabstop>dimInputRectangle</tabstop>
  <tabstop>distInputRect</tabstop>
  <tabstop>dimConnectionRect</tabstop>
  <tabstop>distConnRect</tabstop>
  <tabstop>nrOfMapsFirstLevel</tabstop>
  <tabstop>nrOfMapsSecondLevel</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>nrOfIterations</tabstop>
  <tabstop>eta</tabstop>
  <tabstop>mseError</tabstop>
  <tabstop>momentum</tabstop>
  <tabstop>inputFile</tabstop>
  <tabstop>activationTemp</tabstop>
  <tabstop>trainningButton</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>outputDisplay</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
